### PR TITLE
PathNet: Evolution Channels Gradient Descent in Super Neural Networks

### PR DESCRIPTION
**Pull request by jaesik817**
I implemented PathNet with Tensorflow, which is to learn a huge neural networks with sparse activation and evolution channel selection, https://arxiv.org/abs/1701.08734.

### CODE REVIEW

**Comment by jaesik817** [comment]
I signed it!
